docs/appregistry.md

```diff
@@ -59,179 +57,133 @@ static enableArchitectureIndicator(enabled)
 
 **Parameters:**
 
-| Name                                                     | Type    |
-| -------------------------------------------------------- | ------- |
-| enabled <div class="label basic required">Required</div> | boolean |
+| Name    | Type    | Required |
+| ------- | ------- | -------- |
+| enabled | boolean | yes      |
 
 ---
 
-### `getAppKeys()`
+### `registerConfig()`
 
 ```jsx
-static getAppKeys()
+static registerConfig([config])
 ```
 
-Returns an array of strings.
-
----
+**Parameters:**
 
-### `getRegistry()`
+| Name   | Type      | Required | Description |
+| ------ | --------- | -------- | ----------- |
+| config | AppConfig | yes      | See below.  |
 
-```jsx
-static getRegistry()
-```
+Valid `AppConfig` keys are:
 
-Returns a [Registry](appregistry#registry) object.
+- 'appKey' (string)- Required.
+- 'component' (ComponentProvider) - Optional.
+- 'run' (Function) - Optional.
+- 'section' (boolean) - Optional.
 
 ---
 
-### `getRunnable()`
+### `registerComponent()`
 
 ```jsx
-static getRunnable(appKey)
+static registerComponent(appKey, componentProvider, section?)
```

Comment:
Was wondering why this is changed?

Reply:
I may have resolved the merge conflict incorrectly. I will revert `fix errors in docs` and try again

Reply:
It's an issue with the original commit (not the merging), we'll check the other docs too